battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java

```diff
@@ -119,15 +127,15 @@ private void markEndOfShip() {
     private List<Point2D> getAllNeighborsOf(Point2D point) {
         int x = point.getX();
         int y = point.getY();
-        Point2D[] array = new Point2D[]{
-                Point2D.of(x - 1, y),
-                Point2D.of(x + 1, y),
-                Point2D.of(x, y - 1),
-                Point2D.of(x, y + 1),
-                Point2D.of(x - 1, y - 1),
-                Point2D.of(x + 1, y - 1),
-                Point2D.of(x - 1, y + 1),
-                Point2D.of(x + 1, y + 1)
+        Point2D[] array = new Point2D[] {
+            Point2D.of(x - 1, y),
+            Point2D.of(x + 1, y),
+            Point2D.of(x, y - 1),
+            Point2D.of(x, y + 1),
+            Point2D.of(x - 1, y - 1),
+            Point2D.of(x + 1, y - 1),
+            Point2D.of(x - 1, y + 1),
+            Point2D.of(x + 1, y + 1)
```

Comment:
maybe variable "step", not a lot of "+1" and "-1"

Reply:
ok thanks